lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -805,6 +805,12 @@ sub save_edit :Chained('base') :PathPart('save') :Args(0) {
         }
     }
 
+    $c->ddgcr_post( [ 'ActivityFeed', 'new' ], {
```

Comment:
Cool.  This is a wip so all my comments are unfair.  But I'd say there should probably be a more user friendly way of doing this.